### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)

Some URLs of the docs do not have /ja. Please leave them as they are. (They will be handled on the next translation management system.)

### CODE REVIEW

File: jekyll/_includes/snippets/ja/machine-resource-table.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-| medium  | 2    | 7.5 GB |
-| large   | 4    | 15 GB  |
-| xlarge  | 8    | 32 GB  |
-| 2xlarge | 16   | 64 GB  |
+| クラス     | vCPU | RAM   |
+| ------- | ---- | ----- |
+| medium  | 2    | 100 GB |
+| large   | 4    | 100 GB |
+| xlarge  | 8    | 100 GB |
+| 2xlarge | 16   | 100 GB |
```
Comment (nanophate) [inline jekyll/_includes/snippets/ja/machine-resource-table.md:6]: I think the disk size were added .

```
| クラス     | vCPU | RAM    | ディスクサイズ   |
| ------- | ---- | ------ | ----- | 
| medium  | 2    | 7.5 GB | 100 GB |
| large   | 4    | 15 GB  | 100 GB |
| xlarge  | 8    | 32 GB  | 100 GB |
| 2xlarge | 16   | 64 GB  | 100 GB |

```

https://circleci.com/docs/2.0/executor-types/#using-machine
image

**Review by nanophate — CHANGES_REQUESTED** [review]
I saw that Disk size is added to the table but got replaced with RAM. It would be nice if it can be fixed.

File: jekyll/_includes/snippets/ja/machine-resource-table.md (hunk 1 above)
Comment (Mayuko-Sa) [reply jekyll/_includes/snippets/ja/machine-resource-table.md:6]: Thank you! I'll create a pull-request to fix En version as well!

Comment (Mayuko-Sa) [reply jekyll/_includes/snippets/ja/machine-resource-table.md]: Ah, this has already been fixed in En file, I just have to fix Ja then.

**Review by nanophate — APPROVED** [review]
Thank you for fixing the changes. 🙇 It looks good to me!

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**